src/olympia/blocklist/templates/blocklist/block_submission_change_form.html

```diff
@@ -10,8 +10,10 @@
     {% if can_change_object and change %}
         <input type="submit" value="{% trans 'Update' %}" class="default" name="_save">
     {% endif %}
-    {% if has_signoff_permission and is_pending_signoff %}
+    {% if has_signoff_reject_permission and is_pending_signoff %}
```

Comment:
Should there be a test for this change? I don't see one.

Reply:
https://github.com/mozilla/addons-server/pull/13622/commits/fa46856b102546c8410258626ec4fa193d1ae974